### PR TITLE
Yc search filter

### PR DESCRIPTION
**Pull request by ddm1293**

### CODE REVIEW

**Review by SeePeaJay — APPROVED** [review]

File: frontend/src/components/CreateThread/CreateThreadPetInfo.js
Comment (Huiwen-Zuo) [inline frontend/src/components/CreateThread/CreateThreadPetInfo.js:1]: From lines 28-35, I thought we'd agreed that the Name field is required for both lostPetThread and witnessThread? Lines 10-14 is the useEffect to prefix the petName on both types of threads. I don't think we need  "threadType === 'lostPetThread' &&" part and the previous version is working well.

**Review by Huiwen-Zuo — CHANGES_REQUESTED** [review]
Works well on my pc,

File: frontend/src/components/CreateThread/CreateThreadPetInfo.js
Comment (ddm1293) [reply frontend/src/components/CreateThread/CreateThreadPetInfo.js:1]: i cannot see which lines you were referring to.

Comment (Huiwen-Zuo) [reply frontend/src/components/CreateThread/CreateThreadPetInfo.js:1]: After discussion, I'll change this on my next version of FE. Ok to merge this.

**Review by Huiwen-Zuo — APPROVED** [review]

**Pull request merged**